components/button/buttonStyle.css

```diff
@@ -0,0 +1,11 @@
+.primary-btn button {
+    background-color: #319795;
+    color: white;
```

Comment:
You can make use of CSS variables for bg colors like --primary-bg-color and secondary-bg-color so that it makes the code a bit generic.

Reply:
This one is resolve